packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -1260,7 +1261,7 @@ function emitStdScalar(
         format: {
           propertyName: newScalar?.name ?? ""
         },
-        target: newScalar ?? scalar
+        target: NoTarget
```

Comment:
The scalar type is a raw compiler scalar type and logging this type in our side would occupy the terminal but without any value.